cmd/pebble/cmd_enter.go

```diff
@@ -0,0 +1,188 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/canonical/pebble/internal/logger"
+	"github.com/jessevdk/go-flags"
+)
+
+const shortEnterHelp = "Run pebble command in new pebble environment"
+const longEnterHelp = `
+The enter command behaves like the run command with the possibility of running
+a single pebble command after starting the service manager. It's intended to be
+used as an entrypoint in container images.
+
+When no subcommand is given, the behavior is identical to the run command.
+
+When a subcommand is given, the service manager is started in the background
+without starting "autostart" services (i.e. run --hold, unless --run flag is
+given), and the subcommand is run as a pebble client command that can
+communicate with the service manager instance in the background. The service
+manager is shut down after the commands finishes unless noted otherwise below.
+
+Only a subset of pebble commands is supported as a subcommand:
+
+  exec
+  help
+  ls
+  plan
+  services
+  start
+  version
+
+Furthermore, some commands have restrictions and idiosyncrasies:
+
+  exec, ls, plan, services, version
+    Service manager logging is disabled unless -v/--verbose flag is provided.
```

Comment:
The wording still needs a few style/grammar tweaks I think. But for now, just note that these indentations are removed when displaying the help (i.e., running `pebble enter -h`).

Reply:
Noted, thanks. This is done by `wrapText` function in go-flags. Let's keep it indented and fix go-flags later.